packages/react-components/react-infobutton/src/components/InfoButton/useInfoButtonStyles.ts

```diff
@@ -1,33 +1,166 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { createCustomFocusIndicatorStyle } from '@fluentui/react-tabster';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
+import { tokens } from '@fluentui/react-theme';
 import type { InfoButtonSlots, InfoButtonState } from './InfoButton.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const infoButtonClassNames: SlotClassNames<InfoButtonSlots> = {
+  // This classname is not applied, but it's left here to prevent a linting error.
   root: 'fui-InfoButton',
-  // TODO: add class names for all slots on InfoButtonSlots.
-  // Should be of the form `<slotName>: 'fui-InfoButton__<slotName>`
+  content: 'fui-InfoButton__content',
+  button: 'fui-InfoButton__button',
 };
 
 /**
- * Styles for the root slot
+ * Styles for the button slot
  */
-const useStyles = makeStyles({
-  root: {
-    // TODO Add default styles for the root element
+const useButtonStyles = makeStyles({
+  base: {
+    alignItems: 'center',
+    backgroundColor: tokens.colorTransparentBackground,
+    ...shorthands.borderColor('transparent'),
+    color: tokens.colorNeutralForeground2,
+    display: 'inline-flex',
+    justifyContent: 'center',
+    maxWidth: '28px',
+    minWidth: '28px',
+    outlineStyle: 'none',
+    ...shorthands.padding(tokens.spacingHorizontalXS),
+
+    ':hover': {
+      backgroundColor: tokens.colorTransparentBackgroundHover,
+      ...shorthands.borderColor('transparent'),
+      color: tokens.colorNeutralForeground2BrandHover,
+
+      cursor: 'pointer',
+
+      [`& .${iconFilledClassName}`]: {
+        display: 'inline-flex',
+      },
+      [`& .${iconRegularClassName}`]: {
+        display: 'none',
+      },
+    },
+
+    ':hover:active': {
+      backgroundColor: tokens.colorTransparentBackgroundPressed,
+      ...shorthands.borderColor('transparent'),
+      color: tokens.colorNeutralForeground2BrandPressed,
+
+      outlineStyle: 'none',
+
+      [`& .${iconFilledClassName}`]: {
+        display: 'inline-flex',
+      },
+      [`& .${iconRegularClassName}`]: {
+        display: 'none',
+      },
+    },
+  },
+
+  selected: {
+    color: tokens.colorNeutralForeground2BrandPressed,
+
+    [`& .${iconFilledClassName}`]: {
+      display: 'inline-flex',
+    },
+
+    [`& .${iconRegularClassName}`]: {
+      display: 'none',
+    },
+
+    '@media (forced-colors: active)': {
+      color: 'Highlight',
+    },
+  },
+
+  focusIndicator: createCustomFocusIndicatorStyle({
+    ...shorthands.borderColor(tokens.colorTransparentStroke),
+    ...shorthands.borderRadius(tokens.borderRadiusSmall),
+    outlineColor: tokens.colorTransparentStroke,
+    outlineWidth: tokens.strokeWidthThick,
+    outlineStyle: 'solid',
+    boxShadow: `
+      ${tokens.shadow4},
+      0 0 0 2px ${tokens.colorStrokeFocus2}
```

Comment:
Do we have a token to use in place of `2px` here?

Reply:
I could use a token that has that value, but since it's a spread-radius, I don't think we have a token for that.